app/src/main/res/layout-sw600dp-land/question_player_fragment.xml

```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why do we need tablet & non-tablet landscape layouts for state & question players? It seems like standard landscape is also set up for split screen. Is this meant to fix margins & padding? If so, can we do that via dimens files rather than forking the whole layout?

Reply:
Because there will be some tablets that won't support the split-screen. For these devices, we have some mocks [here](https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/ff2d65ea-38d0-4ce9-a28d-183a03e1b061/).  
>Is this meant to fix margins & padding? If so, can we do that via dimens files rather than forking the whole layout?

Yes, this is meant to fix margins & padding. But I think creating different files for tablet devices will make them easier to develop than using a lot of values in `dimens.xml` for different margins - padding.

Reply:
Resolving this on behalf of Ben.